python/llama-index-server/llama_index/server/api/routers/chat.py

```diff
@@ -22,9 +28,15 @@
 from llama_index.server.api.callbacks.stream_handler import StreamHandler
 from llama_index.server.api.models import ChatRequest
 from llama_index.server.api.utils.vercel_stream import VercelStreamResponse
+from llama_index.server.api.utils.workflow import WorkflowService
 from llama_index.server.services.llamacloud import LlamaCloudFileService
 
 
+class ResumeRequest(BaseModel):
+    chat_id: str
+    response: str
```

Comment:
use might response with a more complex answer (e.g. an image and a text) - in TS we're using `MessageContent`
```suggestion
    response: MessageContent
```